app/models/manager_refresh/inventory/core.rb

```diff
@@ -0,0 +1,56 @@
+module ManagerRefresh::Inventory::Core
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def provider_module
+      @provider_module ||= ManageIQ::Providers::Inflector.provider_module(self)
+    end
+
+    def has_authentications(options = {})
```

Comment:
because `provider_module` is a class method created by `Core` and it's used at class level in the persister...